### DISCUSSION TITLE
Review of my project

### DISCUSSION THREAD

**Discussion by LePaVuki** [LePaVuki/angular-react-DevOps-case#2] [General]
I created a formal PR in the original [angular-react-DevOps-case](https://github.com/ganatan/angular-react-starter) repository

Asking to review my code

@dmitry-mightydevops
@kseniyashaydurova

**Comment by dmitry-mightydevops** [comment]
@LePaVuki you need to open PR in this repo and add both of us as reviewers

**Reply by LePaVuki** [reply]
@dmitry-mightydevops created PR in this repo, added you as the reviewer
